cromwell_tools/tests/test_cromwell_api.py

```diff
@@ -55,7 +55,7 @@ def set_up_auth(self, mock_header):
         auth_options = (
             CromwellAuth.harmonize_credentials(**auth),
             CromwellAuth.harmonize_credentials(**{"secrets_file": secrets_file}),
-            CromwellAuth.harmonize_credentials(**{"caas_key": caas_key_file, "url": url})
+            CromwellAuth.harmonize_credentials(**{"service_account_key": service_account_key, "url": url})
```

Comment:
Existing code, but... is there a benefit to passing `**{"service_account_key": service_account_key, "url": url}` instead of `service_account_key=service_account_key, url=url`?

Reply:
I don't see any benefit for this line, but it seems beneficial to line 56, `CromwellAuth.harmonize_credentials(**auth)` is less verbose than `CromwellAuth.harmonize_credentials(url=url, username=username, password=password)`. So I think keeping the above 3 lines in a consistent style might be a small advantage?